src/core/src/Context.cpp

```diff
@@ -231,6 +233,13 @@ void Context::processPropertyChanged(const ChangedPrim& changedPrim) {
                     tilesetsToReload.emplace(tileset);
                 }
             }
+        } else if (
```

Comment:
Lots of conditional nesting here. I think the outer conditional is a good candidate for a switch with branching functions for readability, as long as the additional call doesn't cause too much overhead.